frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -402,4 +402,49 @@ describe("scenarios > question > filter", () => {
       cy.findByText("Rustic Paper Wallet"); // Product ID 1, Gizmo
     });
   });
+
+  it.skip("should not drop aggregated filters (metabase#11957)", () => {
+    const AGGREGATED_FILTER = "Count is less than or equal to 20";
+
+    cy.request("POST", "/api/card", {
+      name: "11957",
+      dataset_query: {
+        database: 1,
+        query: {
+          "source-query": {
+            "source-table": ORDERS_ID,
+            filter: [">", ["field-id", ORDERS.CREATED_AT], "2020-01-01"],
```

Comment:
@flamber changed the first filter to show all dates after January 1, 2020 - to make sure there's always some result displayed in the table. Not essential for the test, but still nice, imo.

Had we used "current year", it would show "No Results" in 2021 and onward :)